src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -100,12 +104,18 @@ const errorDetailsMap = {
   190: {
     460: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat({
+        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.ACCESS_TOKEN_EXPIRED,
+      })
       .setMessage(
         'The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons',
       )
       .build(),
     default: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat({
+        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.UNAUTHORIZED,
```

Comment:
Can you explain why we are using `UNAUTHORIZED` here ?

Reply:
imported from integrations-lib and used same tag here